rs/tests/testing_verification/testnets/from_config.rs

```diff
@@ -1,3 +1,67 @@
+// Set up a testnet from json files, mostly useful for scripting and automating outside of system tests
```

Comment:
Better place for this would probably be `rs/tests/ict/README`. No?

Reply:
Tried to follow how its done in other testnets, I can add it on also there if needed?

Reply:
I'm just wondering if this will reach the same awareness as it would if it would be in some README. Let's leave it like this and you or @basvandijk can decide whether some README is needed or not.

Reply:
Agreed! Thanks